templates/buildkite-elastic.yml

```diff
@@ -126,6 +127,11 @@ Parameters:
     Default: t2.nano
     MinLength: 1
 
+  EBSOptimized:
+    Description: Optional - Enabled EBSOptimized for the instance
```

Comment:
@lox try launch a spot m4 instance use this template and see if it is set to true. It is not set for us.